ui/apps/platform/src/hooks/useClipboardCopy.ts

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+
+export type useClipboardCopyReturn = {
+    wasCopied: boolean;
+    copyToClipboard: (text: string) => Promise<void>;
+};
+
+/**
+ * Hook that provides shorthand for copying text to the browser's clipboard
+ */
+export default function useClipboardCopy() {
```

Comment:
I don't think the type was being used. 

```suggestion
export default function useClipboardCopy(): UseClipboardCopyReturn {
```

Reply:
Oh good catch! 🫠